opentelemetry/src/OpenTelemetry/Eventlog.hs

```diff
@@ -13,11 +13,17 @@ module OpenTelemetry.Eventlog
     setTag,
     addEvent,
     setParentSpanContext,
+    mkCounter,
+    mkUpDownCounter,
+    mkValueRecorder,
+    mkSumObserver,
+    mkUpDownSumObserver,
+    mkValueObserver,
     add,
     record,
     observe,
     SpanInFlight (..),
-    module OpenTelemetry.Metrics
+    module OpenTelemetry.Metrics_Internal
```

Comment:
Are you sure you want to re-export the whole `Metrics_Internal` here? The `OpenTelemetry.Eventlog` is supposed to be the public API.

Reply:
Ah you're right. I especially shouldn't be re-exporting the `Instrument` constructors since you're supposed to only use `mk<Instrument>`. Fixing.

Reply:
Resolved in 2973326